File: iperf/src/main/java/com/util/iperf/IperfException.java
package com.util.iperf;

/**
 * Created by herman on 6/3/2017.
 */

public class IperfException extends Exception {

    public IperfException() {
    }

    public IperfException(String message) {
        super(message);
    }

    public IperfException(String message, Throwable cause) {
        super(message, cause);
    }

    public IperfException(Throwable cause) {
        super(cause);
    }

    public IperfException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: iperf/src/main/java/com/synaptictools/iperf/IPerfException.kt
package com.synaptictools.iperf

import java.lang.Exception

/**
 * Created by herman on 6/3/2017.
 */
class IPerfException : Exception {
    constructor() {}
    constructor(message: String?) : super(message) {}
    constructor(message: String?, cause: Throwable?) : super(message, cause) {}
    constructor(cause: Throwable?) : super(cause) {}
}